templates/oioswift-proxy-server.conf.erb

```diff
@@ -25,7 +25,7 @@ auto_storage_policies=<%= @auto_storage_policies %>
 <% end -%>
 
 [pipeline:main]
-pipeline = catch_errors <% if @middleware_hashedcontainer -%>proxy-logging hashedcontainer<% end -%> <% if @middleware_gatekeeper -%>gatekeeper<% end -%> <% if @middleware_healthcheck -%>healthcheck<% end -%> proxy-logging cache bulk tempurl proxy-logging <% if @middleware_ratelimit -%>ratelimit <% end -%><% if @auth_system == 'keystone' -%>authtoken <% end -%><% if @middleware_swift3 -%>swift3 <% end -%><% if @auth_system == 'keystone' -%>s3token <% end -%><%= @auth_filter -%> <% if @middleware_regexcontainer -%> proxy-logging regexcontainer <% end -%> <% if @middleware_staticweb -%> staticweb <% end -%> copy container-quotas account-quotas slo dlo <% if @middleware_containerhierarchy -%> proxy-logging container_hierarchy <% end -%> versioned_writes proxy-logging proxy-server
+pipeline = catch_errors <% if @middleware_hashedcontainer -%>proxy-logging hashedcontainer<% end -%> <% if @middleware_gatekeeper -%>gatekeeper<% end -%> <% if @middleware_healthcheck -%>healthcheck<% end -%> proxy-logging cache bulk tempurl proxy-logging <% if @middleware_ratelimit -%>ratelimit <% end -%><% if @auth_system == 'keystone' -%>authtoken <% end -%><% if @middleware_swift3 -%>swift3 <% end -%><% if @auth_system == 'keystone' or @auth_system == 's3only' -%>s3token <% end -%><%= @auth_filter -%> <% if @middleware_regexcontainer -%> proxy-logging regexcontainer <% end -%> <% if @middleware_staticweb -%> staticweb <% end -%> copy container-quotas account-quotas slo dlo <% if @middleware_containerhierarchy -%> proxy-logging container_hierarchy <% end -%> <% if @middleware_versionning -%> versioned_writes <% end -%> proxy-logging proxy-server
```

Comment:
`versioned_writes` middleware is added automatically if not present in the pipeline.

Reply:
ok